src/scoping-shim.js

```diff
@@ -63,8 +64,12 @@ export default class ScopingShim {
     let cssText = [];
     for (let i = 0; i < styles.length; i++) {
       let s = styles[i];
-      cssText.push(s.textContent);
       s.parentNode.removeChild(s);
+      if (!nativeShadow && isUnscopedStyle(s)) {
```

Comment:
If this is `nativeShadow` we should simply leave the style here and not collect it. Apply-shim on its own should do something similar.